src/hal/src/image.rs

```diff
@@ -246,10 +246,11 @@ pub const CUBE_FACES: [CubeFace; 6] = [
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Kind {
-    /// A single row of texels.
+    /// A single one-dimensional row of texels.
```

Comment:
Tbh, I've never seen two-dimentional rows.

Reply:
Yeah but the language used for that section in general is a little squirrelly and I'm trying to make it consistent (maybe unsuccessfully).  We have three "axes" this type can vary on: Dimension (1,2,3), array/not array, and cube/not cube, with dimension only valid for not-cube.